From 486e7d60c520202627995e019a45bfef894c8b1d Mon Sep 17 00:00:00 2001
From: Alexandre Torgue <alexandre.torgue@foss.st.com>
Date: Mon, 21 Dec 2020 15:45:34 +0100
Subject: [PATCH 0063/1141] arm64: defconfig: add addons and cleanup fragments

Add cleanup fragment to reduce build time.
Add addons fragment to add config not yet pushed or ones needed as built-in
instead of module.

Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I59f4c2e1dfa274c7b78305d68630767ed33e6b3a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/190001
Tested-by: Alexandre TORGUE <alexandre.torgue@st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@st.com>
Reviewed-by: CITOOLS <smet-aci-reviews@lists.codex.cro.st.com>
---
 .../fragment-01-defconfig-cleanup.config      | 45 +++++++++++++++++++
 .../fragment-02-defconfig-addons.config       |  1 +
 2 files changed, 46 insertions(+)
 create mode 100644 arch/arm64/configs/fragment-01-defconfig-cleanup.config
 create mode 100644 arch/arm64/configs/fragment-02-defconfig-addons.config

--- /dev/null
+++ b/arch/arm64/configs/fragment-01-defconfig-cleanup.config
@@ -0,0 +1,45 @@
+#  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-01-defconfig-cleanup.config -o fragment-01-defconfig-cleanup.config
+# CONFIG_ARCH_ACTIONS is not set
+# CONFIG_ARCH_ALPINE is not set
+# CONFIG_ARCH_APPLE is not set
+# CONFIG_ARCH_BCM is not set
+# CONFIG_ARCH_BCM2835 is not set
+# CONFIG_ARCH_BCMBCA is not set
+# CONFIG_ARCH_BCM_IPROC is not set
+# CONFIG_ARCH_BERLIN is not set
+# CONFIG_ARCH_BRCMSTB is not set
+# CONFIG_ARCH_EXYNOS is not set
+# CONFIG_ARCH_HISI is not set
+# CONFIG_ARCH_INTEL_SOCFPGA is not set
+# CONFIG_ARCH_K3 is not set
+# CONFIG_ARCH_KEEMBAY is not set
+# CONFIG_ARCH_LAYERSCAPE is not set
+# CONFIG_ARCH_LG1K is not set
+# CONFIG_ARCH_MEDIATEK is not set
+# CONFIG_ARCH_MESON is not set
+# CONFIG_ARCH_MVEBU is not set
+# CONFIG_ARCH_MXC is not set
+# CONFIG_ARCH_NPCM is not set
+# CONFIG_ARCH_NXP is not set
+# CONFIG_ARCH_QCOM is not set
+# CONFIG_ARCH_RENESAS is not set
+# CONFIG_ARCH_ROCKCHIP is not set
+# CONFIG_ARCH_S32 is not set
+# CONFIG_ARCH_SEATTLE is not set
+# CONFIG_ARCH_SPRD is not set
+# CONFIG_ARCH_SUNXI is not set
+# CONFIG_ARCH_SYNQUACER is not set
+# CONFIG_ARCH_TEGRA is not set
+# CONFIG_ARCH_TESLA_FSD is not set
+# CONFIG_ARCH_THUNDER is not set
+# CONFIG_ARCH_THUNDER2 is not set
+# CONFIG_ARCH_UNIPHIER is not set
+# CONFIG_ARCH_VEXPRESS is not set
+# CONFIG_ARCH_VISCONTI is not set
+# CONFIG_ARCH_XGENE is not set
+# CONFIG_ARCH_ZYNQMP is not set
+# CONFIG_MEDIA_ANALOG_TV_SUPPORT is not set
+# CONFIG_MEDIA_DIGITAL_TV_SUPPORT is not set
+# CONFIG_MEDIA_RADIO_SUPPORT is not set
+# CONFIG_MEDIA_SDR_SUPPORT is not set
+# CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
--- /dev/null
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -0,0 +1 @@
+#  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
